vault_password_file

```diff
@@ -0,0 +1,11 @@
+#/bin/bash
```

Comment:
Should this not be `#!/usr/bin/env bash`?

Reply:
+1